src/WasmParser.ts

```diff
@@ -1865,7 +1865,7 @@ declare class TextDecoder {
   public decode(bytes: Uint8Array): string;
 }
 
-export var bytesToString: (Uint8Array) => string;
+export var bytesToString: (Uint8Array: any) => string;
```

Comment:
It shall be `export var bytesToString: (bytes: Uint8Array) => string;`. Thank you for looking into this.